### PR TITLE
Add instructions for using abilities within an Engine

### PR DESCRIPTION
**Pull request by esbanarango**
This PR adds instructions on how to use abilities within an engine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,12 +287,12 @@ test resolver how to resolve abilities from the file structure.
...
 An ability unit test will be created each time you generate a new ability via
 `ember g ability <name>`. The package currently supports generating QUnit and
-Mocha style tests.  
+Mocha style tests.
 
 ### Unit testing in your app
 
 To unit test modules that use the `can` helper, you'll need to explicitly add `needs` for the
-ability and helper file like this:  
+ability and helper file like this:
```
Comment (blimmer) [inline README.md]: Looks like your editor did this and it broke the spacing here - you should be able to just place the "```" on its own line to get the same behavior.

**Comment by esbanarango** [comment]
@blimmer fixed. 👍

**Comment by esbanarango** [comment]
@rlivsey Let me know how this look 👍

**Comment by esbanarango** [comment]
@denzo @blimmer PR comments addressed.

File: README.md [hunk 2]
```diff
@@ -269,6 +269,15 @@ export default Ember.Controller.extend({
 });
 ```
 
+## Accessing abilities within an egine
```
Comment (Exelord) [inline README.md]: Typo :) `egine` -> `engine`.
I would also change it to `Ember engine`

**Comment by esbanarango** [comment]
@Exelord 🙈typo fixed 👍

**Pull request merged**